src/objects/oo/zcl_abapgit_oo_base.clas.abap

```diff
@@ -261,7 +277,36 @@ CLASS zcl_abapgit_oo_base IMPLEMENTATION.
   ENDMETHOD.
 
 
-  METHOD zif_abapgit_oo_object_fnc~update_descriptions.
+  METHOD zif_abapgit_oo_object_fnc~syntax_check.
+    ASSERT 0 = 1. "Subclass responsibility
+  ENDMETHOD.
+
+
+  METHOD zif_abapgit_oo_object_fnc~update_descriptions_class.
+    DATA lt_descriptions LIKE it_descriptions.
+
+    FIELD-SYMBOLS <ls_description> LIKE LINE OF it_descriptions.
+
+    IF it_descriptions IS INITIAL.
+      RETURN.
+    ENDIF.
+
+    " Make sure we keep main language
+    SELECT * FROM seoclasstx INTO TABLE lt_descriptions
+      WHERE clsname = is_key-clsname AND langu = iv_language
+      ORDER BY PRIMARY KEY.
+
+    LOOP AT it_descriptions ASSIGNING <ls_description> WHERE langu <> iv_language.
+      <ls_description>-clsname = is_key-clsname.
```

Comment:
For me it is dumping right here (when pulling https://github.com/abapGit-tests/INTF_i18n)
<img width="817" alt="Screenshot 2024-07-02 at 16 08 15" src="https://github.com/abapGit/abapGit/assets/38354196/88c6b98b-34e7-41ef-9cb1-c142dff5d2b1">

Reply:
Fixed